src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,55 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static nature() {
+    this.alive = this.alive.filter((dead) =>
+      dead.health !== 0);
+  }
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+  }
 }
+Animal.alive = [];
 
 class Herbivore extends Animal {
   // write your code here
+  constructor(name, health = 100) {
```

Comment:
```suggestion
  constructor(name, health) {
```
you don't need to do it every time - you will do it in super constructor